config/maintainers.json

```diff
@@ -2,12 +2,12 @@
   "maintainers": [
     {
       "github_username": "rchavarria",
-      "show_on_website": false,
-      "alumnus": false,
-      "name": null,
-      "bio": null,
-      "link_text": null,
-      "link_url": null,
+      "show_on_website": true,
+      "alumnus": true,
```

Comment:
Hey @rchavarria,

@matthewmorgan is right. `alumnus` is meant to identify a maintainer that has been away for some time - and that's not really your case 😅 so what about making it `false` again?

Reply:
Whoops 😅

I'll update it.